one-line-benchmark/olb.sh

```diff
@@ -0,0 +1,85 @@
+# DiRAC one line benchmark
+
+
+check_os(){
+  # Check that this is running on a supported OS
+
+  if [ "$(eval "uname")" != "Linux" ]; then
+  echo "Cannot run the benchmark on anything but a Linux OS"
+  exit 1
+fi
+}
+
+
+ensure_conda_exists(){
+  # Install conda if it does not exist
+
+  if ! command -v conda &> /dev/null ; then
+    install_conda
+  fi
+}
+
+
+install_conda () {
+  # Install conda using miniforge
+
+  arch=$(eval "uname -m")
```

Comment:
Also here: why not
```suggestion
  arch=$(uname -m)
```
?